crates/loro/src/lib.rs

```diff
@@ -264,6 +264,16 @@ impl LoroDoc {
         self.doc.import_with(bytes, origin.into())
     }
 
+    /// Import json-string.
+    pub fn import_json(&self, json: &str) -> Result<(), LoroError> {
```

Comment:
I prefer `import_json_updates` & `export_json_updates`. `export_json` is ambiguous, which might mean export the json document